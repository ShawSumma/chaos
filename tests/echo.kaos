echo 77
echo "hello world"
echo 'hello world'
str hello = "world"
echo hello

// String literal with escape sequences
echo "Hello\n\tWorld!"
