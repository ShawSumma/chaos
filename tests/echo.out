77hello worldhello worldworldHello
	World!